content/variables/environment-variable-groups.md

```diff
@@ -28,6 +28,32 @@ workflows:
 
 Variables defined in environment variable groups work exactly as [Environment Variables](../building/environment-variables/#using-environment-variables). The value of a variable named `API_TOKEN` can be referenced in a workflow as `$API_TOKEN`. Variables defined with the **_secure_** option will have values obfuscated in the Codemagic UI.
 
+## Encrypting values
+
+Putting values in the Variable value input and marking "Secure" checkbox will automatically encrypt those values. However, file encryption requires an additional base64 file encoding step and the process is done with the help of different machine specific command lines.
```

Comment:
```suggestion
Entering values in the Variable value input and marking the **Secure** checkbox will automatically encrypt those values. However, note that in order to store files as secure environment variables, the file needs to be base64 encoded first. This can be done with the help of different OS-specific command lines.
```